lib/block-supports/dimensions.php

```diff
@@ -38,79 +42,39 @@ function gutenberg_register_dimensions_support( $block_type ) {
  * @param WP_Block_Type $block_type       Block Type.
  * @param array         $block_attributes Block attributes.
  *
- * @return array Block spacing CSS classes and inline styles.
+ * @return array Block dimensions CSS classes and inline styles.
  */
 function gutenberg_apply_dimensions_support( $block_type, $block_attributes ) {
```

Comment:
PHP linting is complaining that `$block_attributes` is unused, but aside from that this LGTM!